packages/codemods/src/codeMods/__tests__/personaToAvatar/componentMod.test.ts

```diff
@@ -0,0 +1,77 @@
+import { Project, SyntaxKind, InterfaceDeclaration, TypeReferenceNode, PropertyAccessExpression } from 'ts-morph';
+import {
+  replacePersonaImport,
+  replaceIPersonaPropsImport,
+  replacePersonaSizeImport,
+} from '../../mods/PersonaToAvatar/PersonaToAvatar.mod';
+import { findJsxTag } from '../../utilities';
+const personaPath = '/**/__tests__/mock/**/persona/**/*.tsx';
+
+describe('Persona component mod tests', () => {
+  let project: Project;
+
+  beforeEach(() => {
+    project = new Project();
+    project.addSourceFilesAtPaths(`${process.cwd()}${personaPath}`);
+  });
+
+  it('can replace persona with avatar', () => {
+    const file = project.getSourceFileOrThrow('mFunction.tsx');
+    replacePersonaImport(file);
+    file.getImportDeclarations().forEach(imp => {
+      imp.getNamedImports().forEach(name => {
+        expect(name.getText()).not.toEqual('Persona');
+      });
+      expect(findJsxTag(file, 'Persona').length).toBe(0);
+    });
+  });
+
+  it('can replace IPersonaProps with AvatarProps', () => {
+    const file = project.getSourceFileOrThrow('mInterface.tsx');
+    replaceIPersonaPropsImport(file);
+    file.getImportDeclarations().forEach(imp => {
+      imp.getNamedImports().forEach(name => {
+        expect(name.getText()).not.toEqual('IPersonaProps');
+      });
+      file.forEachDescendant(val => {
+        switch (val.getKind()) {
+          case SyntaxKind.InterfaceDeclaration: {
+            const tVal = val as InterfaceDeclaration;
+            const struct = tVal.getStructure();
+            expect(
+              (struct.extends as string[])?.some(str => {
```

Comment:
What happens if `struct.extends` isn't defined?

Reply:
It has to be defined, at least that's a guarantee when you call getStructure()

Reply:
In that case can you use `!` instead of `?`